### PR TITLE
[Cherry-pick] Fix hub ui configmap not getting created properly

### PR DESCRIPTION
**Pull request by piyush-garg**
This will fix the hub ui configmap not getting created properly
as it is part of the installerset now and we need to add
transformers and also update based on the cr spec hash

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
Fix hub ui configmap not getting created properly
```

### CODE REVIEW

**Comment by PuneetPunamiya** [comment]
LGTM

**Comment by PuneetPunamiya** [comment]
Thanks for the patch 👍🏻 🤗

File: pkg/reconciler/kubernetes/tektonhub/tektonhub.go [hunk 1]
```diff
@@ -200,14 +200,7 @@ func (r *Reconciler) handleError(err error, th *v1alpha1.TektonHub) error {
 }
 
 func (r *Reconciler) manageUiComponent(ctx context.Context, th *v1alpha1.TektonHub, hubDir, version string) error {
-	if err := r.validateUiConfigMap(ctx, th); err != nil {
-		th.Status.MarkUiDependencyMissing(fmt.Sprintf("UI config map not present: %v", err.Error()))
-		return v1alpha1.REQUEUE_EVENT_AFTER
-	}
-
-	th.Status.MarkUiDependenciesInstalled()
```
Comment (PuneetPunamiya) [inline pkg/reconciler/kubernetes/tektonhub/tektonhub.go:208]: I think we need to mark the status of dependencies as true to make tekton hub status as true

Comment (piyush-garg) [reply pkg/reconciler/kubernetes/tektonhub/tektonhub.go:208]: there is no dependency as such, i removed the condition

**Review by savitaashture — COMMENTED** [review]
/lgtm

:tada: :+1:

**Pull request merged**